contract/models/contract_line.py

```diff
@@ -664,7 +664,7 @@ def _get_period_to_invoice(
         self.ensure_one()
         first_date_invoiced = False
         if not recurring_next_date:
-            return first_date_invoiced, last_date_invoiced, recurring_next_date
+            return first_date_invoiced, False, recurring_next_date
```

Comment:
It would be more readable to `return False, False, False`

Reply:
yes

Reply:
```suggestion
            return False, False, False
```